src/components/navigation/Navbar.tsx

```diff
@@ -1,12 +1,25 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import "./Navbar.scss";
 
 export const Navbar: React.FunctionComponent = () => {
   return <nav>
-    <ul>
-      <li><Link to="/">Home</Link></li>
-      <li><Link to="/rovers">Rovers</Link></li>
-      <li><Link to="/quiz">Quiz</Link></li>
-    </ul>
+
+    <div className="emptySpacingBox"></div>
```

Comment:
Having empty elements just to add space isn't very accessible. Could you achieve the same effect by using a top margin, or setting the position of the element directly?

Reply:
Have left this out for individual pages to deal with